companion/src/mdichild.cpp

```diff
@@ -87,6 +89,22 @@ void MdiChild::refresh(bool expand)
   updateTitle();
 }
 
+void MdiChild::onItemSelected(QModelIndex idx)
+{
+  emit copyAvailable(isModel(idx) ? true : false);
```

Comment:
Well if you're not going to use `viableModelSelected()` it can probably be removed or commented since it's not used otherwise.  The old usage was in modelslist.cpp and is now behind both an `#if 0` and also comments.  But I'd vote for using it.